tests/symmetry/test_groups.py

```diff
@@ -217,6 +219,11 @@ def test_repr(self):
             symbol = ORDERED_SYMBOLS[num - 1]
             assert repr(sg) in f"SpaceGroup({symbol=})"
 
+    def test_valid_symbol(self):  # Added after issue #3845
+        for num in range(1, 231):
+            symbol = SpaceGroup.from_int_number(num).symbol
+            assert SpaceGroup(symbol).int_number == num
```

Comment:
this is definitely a good test to have and an argument in favor of the changes in this PR